src/components/Dashboard/RewardsModule.js

```diff
@@ -162,104 +185,129 @@ const DisputesFees = ({
+          align-items: center;
+        `}
+      >
+        <span
+          css={`
+            color: ${theme.help};
+            margin-right: ${1 * GU}px;
           `}
         >
-          <span
-            css={`
-              color: ${theme.help};
-              margin-right: ${1 * GU}px;
-            `}
-          >
-            Distribution
-          </span>
-          <Help
-            hint="Rewards per dispute"
-            css={`
-              padding: 0;
-            `}
-          >
-            <DisputesFeeDistribution
-              totalDisputesFees={totalDisputesFees}
-              symbol={symbol}
-              decimals={decimals}
-            />
-          </Help>
-        </div>
-      </FeeSection>
-      <FeeSection>
-        <div>
-          <h3
-            css={`
-              ${textStyle('label1')}
-              color: ${theme.surfaceContentSecondary};
-              margin-bottom: ${1 * GU}px;
-            `}
-          >
-            Total
-          </h3>
-          <RowFee
-            label="Total rewards"
-            amount={totalFeesFormatted}
+          Distribution
+        </span>
+        <Help
+          hint="Rewards per dispute"
+          css={`
+            padding: 0;
+          `}
+        >
+          <DisputesFeeDistribution
             symbol={symbol}
-            showPositive
-            css={`
-              margin-bottom: ${2 * GU}px;
-            `}
+            decimals={decimals}
+            distribution={distribution}
           />
-          <Button mode="positive" type="submit" wide>
-            Claim rewards
-          </Button>
-        </div>
-      </FeeSection>
-    </form>
+        </Help>
+      </div>
+    </FeeSection>
+  )
+}
+
+function TotalFees({ totalFees, treasuryBalance }) {
+  const theme = useTheme()
+  const { feeToken } = useCourtConfig()
+  const { symbol, decimals } = feeToken
+
+  const totalFeesFormatted = formatTokenAmount(totalFees, true, decimals, true)
+
+  // We'll show the info section in the case that the account has settlements to do
+  const showInfoSection = !totalFees.eq(treasuryBalance)
+
+  return (
+    <FeeSection>
+      <div>
+        <h3
+          css={`
+            ${textStyle('label1')};
+            color: ${theme.surfaceContentSecondary};
+            margin-bottom: ${1 * GU}px;
+          `}
+        >
+          Total
+        </h3>
+        <RowFee
+          label="Total rewards"
+          amount={totalFeesFormatted}
+          symbol={symbol}
+          showPositive
+          css={`
+            margin-bottom: ${2 * GU}px;
+          `}
+        />
+        <Button
+          mode="positive"
+          type="submit"
+          wide
+          css={`
+            margin-bottom: ${showInfoSection ? 2 * GU : 0}px;
+          `}
+        >
+          Claim rewards
+        </Button>
+        {showInfoSection && (
+          <Info>
+            This action requires multiple transactions to be signed in Metamask.
```

Comment:
```suggestion
            This action requires multiple transactions to be signed by your account.
```

@bpierre Do we have any utilities here or in `use-wallet` that help detect which wallet / provider is being used?

Reply:
Yes we have a function for that in this project! https://github.com/aragon/court-dashboard/blob/58dda7601dd870e2f6ec0bb1f68ca989aaac294b/src/ethereum-providers/index.js#L99

To use it:

```jsx
const { activated } = useWallet()
const provider = getProviderFromUseWalletId(activated)
```